doc/releases/changelog-dev.md

```diff
@@ -166,6 +166,8 @@
 
 <h3>Bug fixes</h3>
 
+* Uses a local random number generator where possible to avoid mutating the global random state.
```

Comment:
```suggestion
* Uses a local random number generator where possible to avoid mutating the global random state.
  [(#3624)](https://github.com/PennyLaneAI/pennylane/pull/3624)
```